src/output.cpp

```diff
@@ -309,7 +309,7 @@ std::string Output::value_to_str(BPFtrace &bpftrace,
   else if (type.IsIntTy()) {
     auto sign = type.IsSigned();
     switch (type.GetIntBitWidth()) {
-      // clang-format off
+        // clang-format off
```

Comment:
Previous formatting looked more correct IMO

Reply:
This is just to make clang format happy. It was a failure before this change.

Reply:
ha, I see